File: app/src/main/java/com/sahibindencourseproject/util/TemperatureUtil.java
package com.sahibindencourseproject.util;

/**
 * Created by Ataer Caner on 2019-12-02.
 * Copyright (c) 2019 sahibinden. All rights reserved.
 */

public class TemperatureUtil {

    public static String getCelcius(Double temp) {
        return Math.round(temp) + "°";
    }
}

File: app/src/main/java/com/sahibindencourseproject/util/TemperatureUtil.kt
package com.sahibindencourseproject.util

/**
 * Created by Ataer Caner on 2019-12-02.
 * Copyright (c) 2019 sahibinden. All rights reserved.
 */

object TemperatureUtil {

    fun getCelcius(temp: Double?): String {
        return Math.round(temp!!).toString() + "°"
    }
}
